lib/scala/kg/src/main/resources/data/test/kgtk/edges.tsv

```diff
@@ -0,0 +1,1000 @@
+node1	relation	node2	node1;label	node2;label	relation;label	relation;dimension	weight	source	origin	sentence	question	id
```

Comment:
Probably want to compress this (bz2) and use the DataResources + compression reader stuff from CskgCsvReader to detect compression.

Reply:
compressed edges.tsv